### DISCUSSION TITLE
How to open a window with capacitor plugin

### DISCUSSION THREAD

**Discussion by anthony-bernardo** [ionic-team/capacitor#3760] [Q&A]
EDIT : posted on stackoverflow https://stackoverflow.com/questions/64693531/new-view-in-ios-with-capacitor-plugin

I cannot open I new empty view 

```
DispatchQueue.main.async {            
            let viewController:AbcdViewController = AbcdViewController()
            self.bridge.viewController.present(viewController, animated: false, completion: nil)
}
```

when I do this, nothing is happening (no error)

I tried with a custom storyboard, but I get error (storyboard not found) : 

```
 let storyBoard = UIStoryboard(name: "Custom", bundle: nil)
 let mainViewController = storyBoard.instantiateViewController(withIdentifier: "AbcdViewController")
 self.bridge.viewController.present(myViewController, animated: true, completion: nil)
```

**Comment by ikeith** [comment]
> ```
> let viewController:AbcdViewController = AbcdViewController()
> self.bridge.viewController.present(viewController, animated: false, completion: nil)
> ```

Where is this being called from?

> ```
>  let storyBoard = UIStoryboard(name: "Custom", bundle: nil)
> ```

Assuming the storyboard exists and is included with your plugin, passing `nil` for the bundle may not be what you need. That defaults to searching in the Application's bundle for the storyboard which is not the same as your plugin's bundle. But that's also dependent on the plugin's `.podspec` file.

Can you share a sample project? That would be much easier to diagnose.
